src/app/login/panel/LoginForm.tsx

```diff
@@ -11,9 +11,9 @@ import {
   DialogActions,
   Snackbar,
 } from "@mui/material";
-import { useForm, FormProvider } from "react-hook-form";
+import { useForm } from "react-hook-form";
 import { login } from "@/apis/auth";
-import SenifitTextField from "../../components/SenifitTextField";
+import SenifitTextField from "../../../components/SenifitTextField";
```

Comment:
```suggestion
import SenifitTextField from "@/components/SenifitTextField";
```
import alias 설정이 되어있어서 이렇게도 처리 가능하니 참고해주세요!